### DISCUSSION TITLE
is there any way i can get this sound to play?

### DISCUSSION THREAD

**Discussion by drversace** [ExistentialAudio/BlackHole#756] [Q&A]
i recorded a very imported video via quick time but unfortunately i didnt select the black hole in the quick time recording options at the time of recording, is there any way i can get that sound into the video.... it is literally crucial to get the sound as i had worked very hard on the meeting and now my audio isnt there....

**Comment by devinroth** [comment]
Did you record the audio anywhere?

Devin

**Reply by drversace** [reply]
no, its stramge zoom audio is not recorded
